samples/openapi3/client/petstore/python-experimental/tests/test_api_validation.py

```diff
@@ -44,6 +46,27 @@ def test_configuration(self):
             config.disabled_client_side_validations = 'foo'
         config.disabled_client_side_validations = ""
 
+    def test_servers(self):
+        config = petstore_api.Configuration(server_index=1, server_variables={'version': 'v1'})
+        api_client = petstore_api.ApiClient(config)
+        api = pet_api.PetApi(api_client)
+
+        def request(expected_url, method, url, **kwargs):
+            assert expected_url == url
+            raise RuntimeError('pass')
+
+        api_client.request = functools.partial(request, 'http://path-server-test.petstore.local/v2/pet')
+        try:
+            api.add_pet({'name': 'pet', 'photo_urls': []})
+        except RuntimeError as e:
+            assert "pass" == str(e)
```

Comment:
Possible improvement:
This works and a more standard python way to do it is to use
[assert_called_with](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_called_with)

Reply:
If we remove support for Python 2, then yes.

Reply:
It's still declared in https://github.com/OpenAPITools/openapi-generator/blob/294d164ab033df0f7c60f38885c9b57c1fb711c3/samples/openapi3/client/petstore/python-experimental/setup.py#L33. I would be happy to change it in the future for Python 3.6+ only.

Reply:
I think we can drop Python 2.x support in the upcoming 5.x release.